gadgets/trace_mount/program.bpf.c

```diff
@@ -117,9 +105,9 @@ static int probe_exit(void *ctx, int ret)
 	if (!argp)
 		return 0;
 
-	eventp = bpf_map_lookup_elem(&heap, &zero);
+	eventp = reserve_buf(sizeof(*eventp));
 	if (!eventp)
-		return 0;
+		goto cleanup;
```

Comment:
Is this fixing an error in the previous implementation? I can't see how it's related to this PR.